File: resources/[essentials]/crp-ui/nui/src/store/modules/cash.js
const state = () => ({
    canShow: false, currentMoney: 0, changedMoney: { status: false, type: 'remove', quantity: 0 }
})

const actions = {
    setMoneyStatus(state, data) {
        state.commit('setMoneyStatus', data);
    },
    setMoney(state, money) {
        state.commit('setMoney', money);
    },
    removeMoney(state, quantity) {
        state.commit('removeMoney', quantity);
    },
    addMoney(state, quantity) {
        state.commit('addMoney', quantity);
    }
}

const mutations = {
    setMoneyStatus(state, data) {
        state.canShow = data.status ? data.status : data;

        if (data.time) {
            setTimeout(() => { state.canShow = false }, data.time);
        }
    },
    setMoney(state, money) {
        state.currentMoney = money, state.canShow = true;

        setTimeout(() => { state.canShow = false }, 15000);
    },
    removeMoney(state, quantity) {
        state.changedMoney = { status: true, type: 'remove', quantity: quantity };
        state.currentMoney = (state.currentMoney - quantity), state.canShow = true;

        setTimeout(() => {
            state.changedMoney.status = false;

            setTimeout(() => { state.canShow = false }, 1000);
        }, 5000);
    },
    addMoney(state, quantity) {
        state.changedMoney = { status: true, type: 'add', quantity: quantity };
        state.currentMoney = (state.currentMoney + quantity), state.canShow = true;

        setTimeout(() => {
            state.changedMoney.status = false;

            setTimeout(() => { state.canShow = false }, 1000);
        }, 5000);
    }
}

export default { namespaced: true, state, actions, mutations }

File: resources/[essentials]/crp-ui/client/modules/cash.lua
RegisterNetEvent('crp-ui:setMoneyStatus')
AddEventHandler('crp-ui:setMoneyStatus', function(data)
	SendNUIMessage({ eventName = 'cash/setMoneyStatus', eventData = data })
end)

RegisterNetEvent('crp-ui:setMoney')
AddEventHandler('crp-ui:setMoney', function(money)
    SendNUIMessage({ eventName = 'cash/setMoney', eventData = money })
end)

RegisterNetEvent('crp-ui:addMoney')
AddEventHandler('crp-ui:addMoney', function(quantity)
    SendNUIMessage({ eventName = 'cash/addMoney', eventData = quantity })
end)

RegisterNetEvent('crp-ui:removeMoney')
AddEventHandler('crp-ui:removeMoney', function(quantity)
    SendNUIMessage({ eventName = 'cash/removeMoney', eventData = quantity })
end)